saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationSensors.scala

```diff
@@ -45,10 +46,11 @@ object EntityRelationSensors {
   }
 
   def relationToFirstArg_MatchingSensor(r: ConllRelation, t: ConllRawToken): Boolean = {
-    r.sentId.equals(t.sentId) && r.e1.wordId == t.wordId
+    r.sentId.equals(t.sentId) && r.e1.wordId == t.wordId && r.e1.hashCode() == t.hashCode()
```

Comment:
Does this change, result in different numbers for the constrained classifiers? If so, shall we update the numbers in the readme?